src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -3796,8 +3796,8 @@ namespace Akka.Pattern
         public Akka.Pattern.CircuitBreaker OnClose(System.Action callback) { }
         public Akka.Pattern.CircuitBreaker OnHalfOpen(System.Action callback) { }
         public Akka.Pattern.CircuitBreaker OnOpen(System.Action callback) { }
-        public async System.Threading.Tasks.Task<T> WithCircuitBreaker<T>(System.Func<System.Threading.Tasks.Task<T>> body) { }
-        public async System.Threading.Tasks.Task WithCircuitBreaker(System.Func<System.Threading.Tasks.Task> body) { }
+        public System.Threading.Tasks.Task<T> WithCircuitBreaker<T>(System.Func<System.Threading.Tasks.Task<T>> body) { }
```

Comment:
async is used as a function modifier to avoid breaking existing code that use await as a variable (and some other reasons). So basically backwards compat reason on a c# feature level. 
Removing or adding it does not influence upstream dependencies at all as far as i know.